src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -76,18 +79,30 @@ private void initBlock()
 
     @NotNull
     @Override
-    public TileEntity createNewTileEntity(World world, int meta)
+    public TileEntity createNewTileEntity(final World world, final int meta)
     {
         //Creates a tile entity for our building
         return new TileEntityColonyBuilding();
     }
 
     /**
-     * Convert the given metadata into a BlockState for this Block
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @Override
+    @Deprecated
+    public boolean isFullBlock(final IBlockState state)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1133)